packages/react/src/components/stepper/Stepper.stories.tsx

```diff
@@ -0,0 +1,663 @@
+
+  const [state, dispatch] = useReducer(reducer, initialState);
+
+  const theme = {
+    '--hds-stepper-color': 'var(--color-tram)',
+    '--hds-step-content-color': 'var(--color-tram)',
+  };
+
+  return (
+    <div>
+      <Stepper
+        theme={theme}
+        labels={labels}
+        language="en"
+        states={state.states}
+        selectedStep={state.activeStep}
+        stepsTotal={5}
+        onStepClick={(event, number) => dispatch({ type: 'setActive', payload: number })}
+      />
+      <div
+        style={{
+          height: '300px',
+          display: 'flex',
+          justifyContent: 'flex-start',
+          alignItems: 'flex-end',
+          gap: '24px',
+          marginLeft: '10px',
+        }}
+      >
+        <Button
+          className="stepper-custom-secondary-button"
+          disabled={state.activeStep === 1}
+          variant="secondary"
+          onClick={() => dispatch({ type: 'setActive', payload: state.activeStep - 1 })}
+          style={{ height: 'fit-content', width: 'fit-content' }}
+          iconLeft={<IconArrowLeft />}
+        >
+          Previous
+        </Button>
+        <Button
+          className="stepper-custom-primary-button"
+          onClick={() => dispatch({ type: 'completeStep', payload: state.activeStep })}
+          style={{ height: 'fit-content', width: 'fit-content' }}
+          iconRight={<IconArrowRight />}
+        >
+          {state.activeStep === 5 ? 'Send' : 'Next'}
+        </Button>
+      </div>
+    </div>
+  );
+};
+
+// args is required for docs tab to show source code
+// eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
+export const SimpleFormExample = (args) => {
+  const activeStepIsValid = (state) => {
+    if (state.activeStep === 1) {
+      // first name
+      return state.fields.firstName.value && state.fields.firstName.value.length > 0;
+    }
+    if (state.activeStep === 2) {
+      // last name
+      return state.fields.lastName.value && state.fields.lastName.value.length > 0;
+    }
+
+    if (state.activeStep === 3) {
+      // age
+      return state.fields.age.value && state.fields.age.value.length > 0;
+    }
+
+    return state.activeStep === 4;
+  };
+
+  const weAreInLastAvailableStep = (state) => {
+    let indexOfLastNonDisabledStep = 0;
+    state.states.forEach((st, index) => {
+      if (st !== 'disabled' && index > indexOfLastNonDisabledStep) {
+        indexOfLastNonDisabledStep = index;
+      }
+    });
+
+    return state.activeStep - 1 === indexOfLastNonDisabledStep;
+  };
+
+  const reducer = (state, action) => {
+    switch (action.type) {
+      case 'changeField': {
+        if (action.newValue.length === 0) {
+          return {
+            activeStep: state.activeStep,
+            states: state.states.map((stateName, index) => {
+              if (index === state.activeStep - 1) {
+                return 'attention';
```

Comment:
When a step is in an erroneous state, the state is set to `attention`. The `attention` was not necessarily meant to convey errors so I would not use it here. Just leave the state to the `available` state in my opinion. We can also ask this in the accessibility review.

Reply:
- [ ] Remove attention state from the example